examples/op_examples/extract.py

```diff
@@ -0,0 +1,16 @@
+from datasets import load_dataset
+
+import lotus
+from lotus.models import LM
+
+lm = LM(model="gpt-4o-mini")
+lotus.settings.configure(lm=lm)
+
+dataset = load_dataset("CShorten/ML-ArXiv-Papers", split="train")
+df = dataset.to_pandas().head(3)
+
+columns = ["problem", "dataset", "results"]
+
+user_instruction = "{abstract}"
+new_df = df.sem_extract(columns, user_instruction)
+print(new_df)
```

Comment:
the output is a bit messy (I see two unamed columns), check how you're reading the data and setting the index

   Unnamed: 0.1  Unnamed: 0  ...                                      dataset_quote                                      results_quote
0             0         0.0  ...  i.i.d. training sample from the joint distribu...  we give an information-theoretic characterizat...
1             1         1.0  ...          "model the network as a random topology;"  "the optimal design improves significantly the...
2             2         2.0  ...  Given a weighted directed acyclic graph whose ...  An algorithm is given whose average cumulative...